lib/cyclonedx/report.rb

```diff
@@ -28,13 +33,22 @@ def to_cyclonedx
         metadata: {},
         components: []
       }
-      cyclonedx_report[:metadata] = converter.build_metadata
 
       # for each scanner report, run the appropriate converter
       @scan_reports.each do |scan_report|
-        cyclonedx_report[:components] << converter(scan_report[0])
+        cyclonedx_report[:components] += converter(scan_report[0])
```

Comment:
Did convertor go from returning an item to returning an array?

Reply:
Right, converter returns an array of components and the final CycloneDX report expects a list of components rather than a list of lists so had to make this change.